src/components/Layout/RightPaneLayout/RightPaneLayout.tsx

```diff
@@ -12,8 +12,15 @@ export function RightPaneLayout({
   paneBgColor?: Palette;
   paneWidth?: number;
 }) {
-  const { isRightPaneOpen, rightPaneContent, clearPane } = useRightPaneContext();
+  const { isRightPaneOpen, rightPaneContent, clearPane, closePane } = useRightPaneContext();
 
+  // Close pane on page unmount
```

Comment:
Question: Just curious, but why do we care about closing the pane if we're unmounting anyway? 

Fwiw I think this would be a good rationale to document w/an in-source comment.

Like right the in-source comment is telling me _what_ you're doing, which is fine, but I can read to the code to answer "what" is happening.

What I may not know, as a maintainer, is _why_ we are doing this, so updating the comment to be like:

```
// Close pane on page unmount because otherwise ...something bad...
```

Reply:
Oh, it is probably "otherwise the next page that has a right pane will show our stale content"?

Reply:
Yes, this is the issue. Comment updated!

Reply:
Great, thanks!